fury/window.py

```diff
@@ -403,6 +406,8 @@ def initialize(self):
 
     def render(self):
         """Render only once."""
+        self.fps = 1.0 / (time.perf_counter() - self._last_render_time)
```

Comment:
It should be `self._fps`?

Reply:
How did I miss that! Thanks for pointing that out @xtanion .